lib/whitehall/document_collection_group_membership_updater.rb

```diff
@@ -0,0 +1,28 @@
+module Whitehall
+  class DocumentCollectionGroupMembershipUpdater
+    attr_reader :document
+
+    def initialize(document)
+      @document = document
+    end
+
+    def call
+      return unless document.published_edition
```

Comment:
I think we should see what we can do to avoid aborting on this if there isn't a live edition.